components/src/css/vars.css.ts

```diff
@@ -86,5 +86,5 @@ createGlobalTheme('[data-theme]', accentVars, accentTokens)
 type ColorVars = typeof modeVars & typeof accentVars
 const colorVars = merge(accentVars, modeVars) as ColorVars
 
-type ThemeVars = typeof baseVars & typeof colorVars
+export type ThemeVars = typeof baseVars & typeof colorVars
```

Comment:
where is this being used?

Reply:
whoops it was but I removed it. will revert